mediagrains/tools/wrap_in_gsf.py

```diff
@@ -172,7 +173,7 @@ def wrap_audio_in_gsf():
         duration = fractions.Fraction(samples_per_grain, sample_rate)
 
         template_grain = AudioGrain(
-            flow_id=flow_id, source_id=source_id, origin_timestamp=args.start_ts, sample_rate=sample_rate,
+            flow_id=flow_id, src=source_id, origin_timestamp=args.start_ts, sample_rate=sample_rate,
```

Comment:
Is it correct this is `src` and not `src_id` like the others?

Reply:
🤦 
No. No it's not. Fixed.